frontend/src/views/Evaluation/Alignment/AlignmentView.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import { Barrier, Evaluation, Question } from '../../../api/models'
+import { Box } from '@material-ui/core'
+import BarrierQuestionsView from '../../../components/BarrierQuestionsView'
+import EvaluationSidebar from '../EvaluationSidebar'
+import { useQuestionsQuery } from '../Preparation/PerparationGQL'
+import { TextArea } from '@equinor/fusion-components'
+import AnswerSummarySidebar from '../../../components/AnswerSummarySidebar'
+
+interface AlignmentViewProps
+{
+    evaluation: Evaluation
+}
+
+const AlignmentView = ({evaluation}: AlignmentViewProps) => {
+    const [selectedBarrier, setSelectedBarrier] = React.useState<Barrier>(Barrier.Gm)
+    const [selectedQuestion, setSelectedQuestion] = React.useState<Question | undefined>(undefined)
+
+    const {loading: loadingQuestions, questions, error: errorQuestions} = useQuestionsQuery(evaluation.id)
+
+    if(errorQuestions !== undefined){
+        return <div>
+            <TextArea
+                value={`Error in loading questions: ${JSON.stringify(errorQuestions)}`}
+                onChange={() => { }}
+            />
+        </div>
+    }
+
+    if(loadingQuestions){
+        return <>Loading...</>
+    }
+
+    if(questions === undefined){
+        return <div>
+            <TextArea
+                value={`Questions is undefined`}
+                onChange={() => { }}
+            />
+        </div>
+    }
+
+    return (
+        <>
+            <Box display="flex" height={1}>
+                <Box>
+                    <EvaluationSidebar
+                        questions={questions}
+                        barrier={selectedBarrier}
+                        onBarrierSelected={ (barrier) => setSelectedBarrier(barrier)}
+                    />
+                </Box>
+                <Box p="20px" width="1">
+                    <BarrierQuestionsView
+                        barrier={selectedBarrier}
+                        questions={questions}
+                        showAnswerSummaryButton={true}
+                        onQuestionSummarySelected={ (question: Question) => {
+                            setSelectedQuestion(question)
+                        }}
+                    />
+                </Box>
+            </Box>
+
+            { selectedQuestion &&
+                <AnswerSummarySidebar
+                    open={selectedQuestion != undefined}
+                    onCloseClick={() => { setSelectedQuestion(undefined) }}
+                    question={selectedQuestion} />
+            }
```

Comment:
Since the form has an `open` prop, we do not need the nesting. Then we will not get messages about unable to render unmounted component, although they are hard to see among all our other warnings. 

```suggestion
            <AnswerSummarySidebar
                open={selectedQuestion != undefined}
                onCloseClick={() => { setSelectedQuestion(undefined) }}
                question={selectedQuestion} 
            />
```